ws-client/src/client.rs

```diff
@@ -602,6 +602,12 @@ async fn background_task(
 					let _ = reg.send_back.send(Err(Error::MethodAlreadyRegistered(reg.method)));
 				}
 			}
+
+			// User called `on_notification` on the front-end.
+			Either::Left((Some(FrontToBack::UnregisterNotification(method)), _)) => {
+				log::trace!("[backend] unregistering notification handler: {:?}", method);
+				let _ = manager.remove_notification_handler(&method);
```

Comment:
I think you can pass this by value instead then you don't have to clone/allocate in `remove_notification`